### ISSUE TITLE
Consistent coloration of lands

### ISSUE THREAD

**Issue by cdanis** [cdanis/urzas-overlay#23]
"stomping ground was gold background but fetchlands were grey"
-- walkingeyerobot, 2017

**Comment by cdanis** [comment]
it would be cool to add proper gradients

**Comment by kprevas** [comment]
oh yeah gradients would be dope

**Comment by 7sempra** [comment]
Yeah, so the logic needs to be adjusted. Currently it looks at color identity, which is obviously wrong. The logic is:

Is basic? => Color that basic produces
Is land? => Brown
Is Multicolored? => Gold
Is Artifact? => Silver
else => Use the monocolor (or silver for colorless since we don't have a colorless frame)

Also I dunno what you mean about gradients. The piping? We don't show any piping on these names atm.

**Comment by cdanis** [comment]
http://magiccards.info/scans/en/gtc/247.jpg

it would be super mega dope if Stomping Ground had a R-->G gradient as its
background.

**Comment by 7sempra** [comment]
THAT'S THE PIPING I JUST SAID WE DON'T SHOW PIPING ERMAHGERD RAGE RAGE RAGE :tttt:

**Comment by cdanis** [comment]
i'm saying we should and it should be the background

looks sweet and communicates useful info

**Comment by 7sempra** [comment]
yeah okay fine

**Comment by cdanis** [comment]
hey 7sempra bro were you gonna work on this?

**Comment by terrycunha** [comment]
.bg-land {
    background: #b1a9a9;
    background: -moz-linear-gradient(left, #b1a9a9 0%, #c7e1cf 97%);
    background: -webkit-linear-gradient(left, #b1a9a9 0%,#c7e1cf 97%);
    background: linear-gradient(to right, #b1a9a9 0%,#c7e1cf 97%);
    filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#b1a9a9', endColorstr='#c7e1cf',GradientType=1 );
    border: 0px solid #000;


<img width="357" alt="screen shot 2017-03-14 at 3 36 41 pm" src="https://cloud.githubusercontent.com/assets/6564403/23919208/5bfe7fc2-08cc-11e7-9cef-5eaca6c17e25.png">
